applications/openshift/scc/scc_limit_container_allowed_capabilities/rule.yml

```diff
@@ -31,26 +31,16 @@ references:
 ocil_clause: 'allowed capabilities listings in SCCs needs review'
 
 ocil: |-
-    Inspect each SCC returned from running the following command:
-    <pre>$ oc get scc</pre>
-    Next, examine the outputs of the following commands:
-    <pre>$ oc describe roles --all-namespaces</pre>
-    <pre>$ oc describe clusterroles</pre>
-    For any role/clusterrole that reference the
-    <tt>securitycontextconstraints</tt> resource with the <tt>resourceNames</tt>
-    of the SCCs that do not list an explicit <tt>allowedCapabilities</tt>, examine the
-    associated rolebindings to account for the users that are bound to the role.
-    Review each SCC and determine that only required capabilities are either
-    completely added as a list entry under <tt>allowedCapabilities</tt>,
-    or that all the un-required capabilities are dropped for containers and SCCs.
-    varible var_sccs_with_allowed_capabilities_regex can be set to exclude certain
-    SCCs from the check.
-    Use following command to verify if the correct regex is being used, this ouput
-    will list unqualified SCCs:
+    This rule checks the SCCs with allowedCapabilities set to non-null
+    and fails if there are more such SCCs than those allowed in the variable
+    named var-sccs-with-allowed-capabilities-regex. To debug the rule,
+    check the variable value, e.g:
+    <pre>$ oc get variable ocp4-var-sccs-with-allowed-capabilities-regex  -ojsonpath='{.value}' </pre>
+    Then use following command to list the SCCs that would fail the test:
     <pre>$ oc get scc -o json | {{{ jqfilter }}}</pre>
-    {{.var_sccs_with_allowed_capabilities_regex}} should be replace to the actual value set,
-    either the default one or the one set from TailoredProfile.
-
+    Please replace the regular expression in the test command ({{.var_sccs_with_allowed_capabilities_regex}})
```

Comment:
We ended up solving this by customizing this variable within a `TailoredProfile`... if we override the variable in the `ProfileBundle` directly, does it not get overridden after an operator upgrade?

Reply:
Ahh, good point. I added that to the description (instructions should only tell you how to evaluate pass/fail). The current description renders like this:
```
Description:   Containers should not enable more capabilites than needed as this opens the door for malicious use. To enable only the required capabilities, the appropriate Security Context Constraints (SCCs) should set capabilities as a 
list in allowedCapabilities. In case an SCC outside the default allow list in the variable var-sccs-with-allowed-capabilities-regex is being flagged, create a TailoredProfile and add the additional SCC to the regular expression in the var
iable var-sccs-with-allowed-capabilities-regex.
```